apps/vr-tests-react-components/src/stories/Badge.stories.tsx

```diff
@@ -38,6 +36,30 @@ const useStyles = makeStyles({
   brandContainer: {
     backgroundColor: tokens.colorBrandBackgroundStatic,
   },
+
+  groupSet: {
+    display: 'inline-flex',
+    flexDirection: 'column',
+    ...shorthands.padding(0, '16px'),
+    rowGap: '16px',
```

Comment:
If we're avoiding `flex-gap` in the component themselves we should probably also avoid it in stories.